app/src/test/java/uk/gov/onelogin/wallet/DeleteWalletDataUseCaseTest.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.onelogin.wallet
+
+import androidx.fragment.app.FragmentActivity
+import io.ktor.util.reflect.instanceOf
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
```

Comment:
Note for the future: Please favour the Kotlin test imports. 
It is a bit subjective, but if we need to swap / downgrade the JUnit version (we actually might!) then we don't have to change any tests as Kotlin test is library agnostic